nntrainer/layers/layer_context.h

```diff
@@ -513,10 +514,18 @@ class RunLayerContext {
    * @brief check current graident is first access
    *
    * @param idx index
-   * @return bool true if last access
+   * @return bool true if first access
```

Comment:
add a note - equivalent to weight last access

Reply:
Done

Reply:
I think it might be better to say last execution order. and first execution order of that gradient or weight.

Reply:
updated to the comment :)